Reconsider the usefulness of Unknown

Pylint is failing on `pandas` because of `Unknown` not having a name attribute.
```
    def visit_call(self, node):
        """Visit a Call node."""
        try:
            for inferred in node.func.infer():
                if inferred is astroid.Uninferable:
                    continue
>               if inferred.root().name == OPEN_MODULE:
E               AttributeError: 'Unknown' object has no attribute 'name'

../pylint/pylint/checkers/stdlib.py:209: AttributeError
```

--- a/astroid/node_classes.py
+++ b/astroid/node_classes.py
@@ -4421,6 +4421,10 @@ class Unknown(mixins.AssignTypeMixin, NodeNG):
     introspection failed.
     """
     name = "Unknown"
+
+    def qname(self):
+        return "Unknown"
+
     def infer(self, context=None, **kwargs):
         """Inference on an Unknown node immediately terminates."""
         yield util.Uninferable

--- a/astroid/node_classes.py
+++ b/astroid/node_classes.py
@@ -4420,6 +4420,7 @@ class Unknown(mixins.AssignTypeMixin, NodeNG):
     the args attribute of FunctionDef nodes where function signature
     introspection failed.
     """
+    name = "Unknown"
     def infer(self, context=None, **kwargs):
         """Inference on an Unknown node immediately terminates."""
         yield util.Uninferable

--- a/astroid/tests/unittest_nodes.py
+++ b/astroid/tests/unittest_nodes.py
@@ -863,6 +863,7 @@ def test_unknown():
     assert isinstance(next(nodes.Unknown().infer()),
                       type(util.Uninferable))
     assert isinstance(nodes.Unknown().name, str)
+    assert isinstance(nodes.Unknown().qname(), str)
 
 
 if __name__ == '__main__':

--- a/astroid/tests/unittest_nodes.py
+++ b/astroid/tests/unittest_nodes.py
@@ -858,5 +858,12 @@ class ContextTest(unittest.TestCase):
         self.assertIs(starred.ctx, astroid.Store)
 
 
+def test_unknown():
+    """Test Unknown node"""
+    assert isinstance(next(nodes.Unknown().infer()),
+                      type(util.Uninferable))
+    assert isinstance(nodes.Unknown().name, str)
+
+
 if __name__ == '__main__':
     unittest.main()

